lldb/test/API/lang/cpp/structured-binding/main.cpp

```diff
@@ -1,13 +1,80 @@
 // Structured binding in C++ can bind identifiers to subobjects of an object.
 //
-// There are three cases we need to test:
+// There are four cases we need to test:
 // 1) arrays
-// 2) tuples like objects
-// 3) non-static data members
+// 2) tuple-like objects with `get` member functions
+// 3) tuple-like objects with `get` free functions
+// 4) non-static data members
 //
 // They can also bind by copy, reference or rvalue reference.
 
-#include <tuple>
```

Comment:
Removed so we can better control what kind of tuple-like decomposition we're testing here.